src/test/java/org/broadinstitute/dsde/consent/ontology/integration/ElasticSearchAutocompleteAPITest.java

```diff
@@ -0,0 +1,178 @@
+package org.broadinstitute.dsde.consent.ontology.integration;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.broadinstitute.dsde.consent.ontology.configurations.ElasticSearchConfiguration;
+import org.broadinstitute.dsde.consent.ontology.resources.model.TermResource;
+import org.broadinstitute.dsde.consent.ontology.service.ElasticSearchAutocompleteAPI;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Integration test classes use the maven integration framework to start up an elastic search instance.
+ * Tests can be run locally if a local docker es instance is manually started up:
+ * docker run -p 9200:9200 -e "xpack.security.enabled=false" -e "http.host=0.0.0.0" -e "transport.host=127.0.0.1" docker.elastic.co/elasticsearch/elasticsearch:5.5.0
+ */
+@Category(IntegrationTest.class)
+public class ElasticSearchAutocompleteAPITest {
+
+    private static ElasticSearchAutocompleteAPI autocompleteAPI;
+    private static final String INDEX_NAME = "ontology-integration";
+    private static final Logger logger = LoggerFactory.getLogger(ElasticSearchAutocompleteAPITest.class);
+    private static Gson gson = new GsonBuilder().create();
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        ElasticSearchConfiguration configuration = new ElasticSearchConfiguration();
+        configuration.setIndex(INDEX_NAME);
+        configuration.setServers(Collections.singletonList("localhost"));
+        final Client client = ClientBuilder.newClient();
+        autocompleteAPI = new ElasticSearchAutocompleteAPI(configuration, client);
+        IndexSupport.createIndex(client, configuration);
+        IndexSupport.populateIndex(configuration);
+    }
+
+    @After
+    public void shutDown() throws Exception {
+    }
+
+    @Test
+    public void testOrchestrationReIndex() throws Exception {
```

Comment:
I'm guessing this stresses the same thing that an orch reindex does ... but perhaps choose a more informative name here, like `testConcurrentLookups` or something?